src/elements/Segment/Segments.js

```diff
@@ -0,0 +1,46 @@
+import React, {Children, Component, PropTypes} from 'react';
+import _ from 'lodash';
+import invariant from 'fbjs/lib/invariant';
+import classNames from 'classnames';
+import META from 'src/utils/Meta';
+import Segment from './Segment';
+
+export default class Segments extends Component {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+  };
+
+  componentDidMount() {
```

Comment:
Tried to have this live in `componentWillMount` but ran into trouble getting tests to pass conformance tests.